src/nodemetadata.cpp

```diff
@@ -39,28 +40,39 @@ NodeMetadata::~NodeMetadata()
 	delete m_inventory;
 }
 
-void NodeMetadata::serialize(std::ostream &os) const
+void NodeMetadata::serialize(std::ostream &os, u8 version, bool disk) const
```

Comment:
While that might be clearer, I named the param this way for consistency with `MapBlock::serialize` ([here](https://github.com/minetest/minetest/blob/master/src/mapblock.cpp#L556)) and would like to keep it that way.